source/compiler/sc3.c

```diff
@@ -1390,6 +1390,14 @@ static int hier2(value *lval)
     while (paranthese--)
       needtoken(')');
     return FALSE;
+  case t__STATIC_ASSERT:    
+    static_check(FALSE);
+    pc_sideeffect=TRUE;
+    return FALSE;
+  case t__STATIC_CHECK:
```

Comment:
Since`static_check` has no underscores, why is its token called `t__STATIC_CHECK`?

Reply:
That was a typo while adding the underscore prefixes.